src/js/pages/attendee/NewsFeed.jsx

```diff
@@ -1,130 +1,210 @@
 import React from "react";
-
 import NewsFeedPost from "components/NewsFeedPost";
 import NewsFeedStore from "stores/NewsFeedStore";
-// import style from 'sass/newsfeed.scss';
+import Dropzone from 'react-dropzone';
+import MdHighlightRemove from 'react-icons/lib/md/highlight-remove';
+import MdPhotoCamera from 'react-icons/lib/md/photo-camera';
+import TiSocialFacebook from 'react-icons/lib/ti/social-facebook';
+import TiSocialTwitter from 'react-icons/lib/ti/social-twitter';
+// import style from '../../sass/newsfeed.scss';
 
+// TODO: LIMIT NUMBER OF NEWLINES?
 
 export default class NewsFeed extends React.Component {
 
-	constructor() {
+  constructor() {
     super();
     this.getNewsFeedPosts = this.getNewsFeedPosts.bind(this);
     this.createPost = this.createPost.bind(this);
     this.twToggle = this.twToggle.bind(this);
     this.fbToggle = this.fbToggle.bind(this);
+    this.onDrop = this.onDrop.bind(this);
+    this.deletePhoto = this.deletePhoto.bind(this);
+
     this.state = {
-      posts: NewsFeedStore.getAll(),
+      contentFeed: {
+        items: [],
+        nextOffset: null,
+      },
+      file: null,
+      preview: null,
       fbPost: false,
       twPost: false,
-      userText: "#SoftwareDemoDay",
+      // TODO: HOW CAN I GET THESE? (If user is integrated into social media channels)
+      twIntegration: true,
```

Comment:
@kyl191 might know more. There was similar discussion around the preferences (since we allow organizers to turn off integrations in the conference settings).